CONTRIBUTING.md

```diff
@@ -27,3 +27,31 @@ for details, such as:
 * Development setup and testing your changes
 * Submitting a pull request and finding a reviewer
 
+To build and install the whole project from the source distribution,
```

Comment:
Per dev@ thread, let's move all concrete technical instructions to the linked contribution guide.